cmd/node/config/config.toml

```diff
@@ -589,6 +589,8 @@
     TrieOperationsDeadlineMilliseconds = 10000
     # GetAddressesBulkMaxSize represents the maximum number of addresses to be fetched in a bulk per API request. 0 means unlimited
     GetAddressesBulkMaxSize = 100
+    # VmQueryDelayAfterStartInSec represents the number of seconds to wait when starting node before accepting vm query requests
+    VmQueryDelayAfterStartInSec = 5
```

Comment:
Isn't 5 sec even for testnets too low for a value?

Reply:
used this for testing, forgot to change back. done